xml/install_caasp_heat_templates.xml

```diff
@@ -563,39 +576,62 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
       admin node bootstrapping steps.
      </para>
      <para>
-      Apply the &caasp; 3.0 Maintenance Update with the following steps:
+      Apply the &caasp; 3.0 Maintenance Update with the following steps,
+      run on each node in the cluster. This can be done one at a time,
```

Comment:
I'm sorry, I must have missed this the first time "steps, run on each node in the cluster." -> please remove the comma!